tests/ui/tests/header.js

```diff
@@ -45,35 +45,54 @@ define([
         },
 
         'The header search box expands and contracts correctly': function() {
+            // Possible ToDo:  Move click() call after the transitionend event listener
 
+            var remote = this.remote;
             var searchBoxId = 'main-q';
             var homeSearchBoxId = 'home-q';
-            var originalSize;
+            var originalWidth;
 
-            return this.remote
+            remote.setExecuteAsyncTimeout(config.asyncExecutionTimeout);
+
+            var transitionEndCallback = function() {
+                return remote.executeAsync(function(done) {
+                    var element = document.getElementById('main-q');
```

Comment:
I know I'm repeating a string we have stored here but this exact JS is being injected into the page so we have to repeat it :/